portfolio/src/main/webapp/materialize.js

```diff
@@ -12412,24 +12412,51 @@ if (dark_mode == "no") {
 
 function createListElement(text) {
   const liElement = document.createElement('li');
-  liElement.innerText = text;
+  text = "    " + text;
+  liElement.innerHTML = "<button onclick=\"deleteComm()\" class=\"btn-small\"><i class=\"material-icons\">cancel</i></button>" + text;
   return liElement;
 }
+
+function createParagraphElem(text) {
+    const pElement = document.createElement('p');
+    pElement.innerText = text;
+    return pElement;
+}
+
+//function deleteComm(comment) {
+//    const params = new URLSearchParams();
+//    params.append('id', )
+//}
+
 var i;
```

Comment:
could you pleaae replace i with meaningful var?

Reply:
Done.